### PR TITLE
Allow 405 Method Not Allowed for DELETE requests in teardowns

### PR DESCRIPTION
**Pull request by markphelps**
Fixes: https://github.com/opencontainers/distribution-spec/issues/179

Allows `405 Method Not Allowed` HTTP Response Status in teardown methods in conformance tests

### CODE REVIEW

**Review by jdolitsky — APPROVED** [review]
👍

**Review by jdolitsky — APPROVED** [review]

**Comment by jdolitsky** [comment]
Hi, just waiting on another LGTM cc @opencontainers/distribution-spec-maintainers

**Comment by markphelps** [comment]
@jdolitsky it doesnt seem that team `@opencontainers/distribution-spec-maintainers` exists anymore, is there perhaps someone individually that can take a look? thanks!

**Review by mikebrow — APPROVED** [review]
LGTM

**Pull request merged**
